packages/website/index.html

```diff
@@ -17,42 +17,19 @@
     />
     <script src="https://cdn.jsdelivr.net/npm/vanillajs-datepicker@1.3.4/dist/js/datepicker-full.min.js"></script>
 
-    <link rel="icon" href="./favicon.ico" type="image/x-icon" />
```

Comment:
modify template to use absolute paths because they will may now be included in subdirectories

are there any relative links we use elsewhere?